README

```diff
@@ -95,6 +95,13 @@ file is $XDG_CONFIG_HOME/Yubico/u2f_keys. If the environment variable
 is not set, $HOME/.config/Yubico/u2f_keys is used.
 (more on <<files,Authorization Mapping Files>>).
 
+lockfile=file::
+Set the location of the file that is used for touch request notifications.
+This file will be opened when pam-u2f starts waiting for a user to touch the device,
+and will be closed when it no longer waits for a touch.
+Use inotify to listen on these events, or a more high-level tool like https://github.com/maximbaz/yubikey-touch-detector[yubikey-touch-detector].
+Default value: /var/run/user/$UID/pam-u2f-touch
+
```

Comment:
Can you add the same blurb to `/man/pam_u2f.8.txt`?
Please don't add the link there :)

Reply:
I think that fine. Thanks.